src/api/api.js

```diff
@@ -112,10 +130,21 @@ async function getMyEvents(start) {
   }
 }
 
+async function getSitewideAnnouncements(paramObj) {
+  try {
+    const params = await objToParams(paramObj);
+    const { data } = await protectedResourceAxios.get(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/announcements${params}`);
```

Comment:
Does this work? It shouldn't lol

The protectedResourceAxios is configured with a base domain, so this would hit DOMAIN/DOMAIN/api/v1...

Reply:
Fixed